frontend/src/scenes/surveys/components/SurveysTable.tsx

```diff
@@ -0,0 +1,425 @@
+import { useActions, useValues } from 'kea'
+import { router } from 'kea-router'
+
+import { LemonButton, LemonDialog, LemonDivider, LemonInput, LemonSelect, LemonTable, Spinner } from '@posthog/lemon-ui'
+
+import { AccessControlAction } from 'lib/components/AccessControlAction'
+import { MemberSelect } from 'lib/components/MemberSelect'
+import { dayjs } from 'lib/dayjs'
+import { More } from 'lib/lemon-ui/LemonButton/More'
+import { LemonTableColumn } from 'lib/lemon-ui/LemonTable'
+import { LemonTableLink } from 'lib/lemon-ui/LemonTable/LemonTableLink'
+import { createdAtColumn } from 'lib/lemon-ui/LemonTable/columnUtils'
+import { cn } from 'lib/utils/css-classes'
+import { ProductIntentContext } from 'lib/utils/product-intents'
+import stringWithWBR from 'lib/utils/stringWithWBR'
+import { SurveyStatusTag } from 'scenes/surveys/components/SurveyStatusTag'
+import { SurveysEmptyState } from 'scenes/surveys/components/empty-state/SurveysEmptyState'
+import { SURVEY_TYPE_LABEL_MAP, SurveyQuestionLabel } from 'scenes/surveys/constants'
+import { SurveysTabs, surveysLogic } from 'scenes/surveys/surveysLogic'
+import { isSurveyRunning } from 'scenes/surveys/utils'
+import { urls } from 'scenes/urls'
+
+import { AccessControlLevel, AccessControlResourceType, Survey } from '~/types'
+
+export function SurveysTable(): JSX.Element {
+    const {
+        data: { surveys },
+        searchedSurveys,
+        dataLoading,
+        surveysResponsesCount,
+        surveysResponsesCountLoading,
+        searchTerm,
+        filters,
+        tab,
+        hasNextPage,
+        hasNextSearchPage,
+    } = useValues(surveysLogic)
+
+    const { deleteSurvey, updateSurvey, setSearchTerm, setSurveysFilters, loadNextPage, loadNextSearchPage } =
+        useActions(surveysLogic)
+
+    const shouldShowEmptyState = !dataLoading && surveys.length === 0
+
+    if (shouldShowEmptyState) {
+        return <SurveysEmptyState numOfSurveys={surveys.length} />
+    }
+
+    return (
+        <>
```

Comment:
this component is just copied from Surveys.tsx, no need to review the actual rendering